ast/ast.go

```diff
@@ -76,6 +78,22 @@ type Operator struct {
 	Val  OperatorVal
 }
 
+// Operator map
+var OperatorTypeMap = map[string]OperatorType{
+	"+":  Plus,
+	"-":  Minus,
+	"!":  Bang,
+	"*":  Asterisk,
+	"/":  Slash,
+	"%":  Mod,
+	"<":  LT,
+	">":  GT,
+	"<=": LTE,
+	">=": GTE,
+	"==": EQ,
+	"!=": NOT_EQ,
+}
```

Comment:
How about move this map into `parse/parser.go` and instead of mapping `string` with `OperatorType`, `TokenType` with `OperatorType`? 

Advantage of this is that when we mapping `TokenType` and `OperatorType`, then this doesn't dependent on `string` value of token anymore

Reply:
Ok~